internal/test/framework/framework.go

```diff
@@ -149,45 +150,49 @@ func (f *TestFramework) setupWinRMClient() error {
 // configureOpenSSHServer configures the OpenSSH server using WinRM client installed on the Windows VM.
 // The OpenSSH server is installed as part of WNI tool's CreateVM method.
 func (f *TestFramework) configureOpenSSHServer() error {
+	stdout := new(bytes.Buffer)
```

Comment:
Are you finding anything annoying in the output to silent it?

Reply:
This is the output on every test run
```
[ssoto@localhost windows-machine-config-operator]$ make run-wsu-ci-e2e-test 
hack/run-wsu-ci-e2e-test.sh
ssoto
network.operator.openshift.io/cluster patched (no change)

Name                           Version          Source           Summary                                               
----                           -------          ------           -------                                               
nuget                          2.8.5.208        https://onege... NuGet provider for the OneGet meta-package manager    




    Directory: C:\


Mode                LastWriteTime         Length Name                                                                  
----                -------------         ------ ----                                                                  
d-----       12/13/2019   2:53 AM                Temp                                                                  


=== RUN   TestWSU
```

Reply:
SG